### PR TITLE
Restore options lost with PR #492

### PR DESCRIPTION
**Pull request by simon-spinner**
As a result partition create is currently broken throwing e.g.,` KeyError: 'ifl-absolute-capping'`.

### CODE REVIEW

File: zhmccli/_cmd_partition.py [hunk 1]
```diff
@@ -368,15 +399,21 @@ def partition_dump(cmd_ctx, cpc, partition, **options):
 @optgroup.option('--access-extended-counter-set', type=bool, required=False,
                  help='Indicates if extended counter set authorization control '
                  'is requested. Default: False')
-@optgroup.option('--access-coprocessor-group-set', type=bool, required=False,
```
Comment (simon-spinner) [inline zhmccli/_cmd_partition.py:371]: This property on the API never had a function. I would propose to remove it from here.

**Comment by andy-maier** [comment]
I have compared the options of partition create/update before PR #492 and as of this PR here, and what is still missing, is:

partition create:
```
mis @option('--boot-timeout'
mis @option('--cp-processing-weight-capped'
mis @option('--ifl-processing-weight-capped'
mis @option('--permit-aes-key-import-functions'
mis @option('--permit-des-key-import-functions'
mis @option('--permit-ecc-key-import-functions'
mis @option('--processor-management-enabled'
mis @option('--reserve-resources'
mis @option('--short-name'
```

partition update:

```
mis @option('--boot-configuration'
mis @option('--boot-ficon-loader-mode'
mis @option('--boot-iso-insfile'
mis @option('--boot-load-parameters'
mis @option('--boot-media-type'
mis @option('--boot-os-specific-parameters'
mis @option('--boot-record-lba'
mis @option('--boot-record-location'
mis @option('--cp-absolute-capping'
mis @option('--cp-processing-weight-capped'
mis @option('--ifl-absolute-capping'
mis @option('--ifl-processing-weight-capped'
mis @option('--partition-id'
mis @option('--permit-aes-key-import-functions'
mis @option('--permit-des-key-import-functions'
mis @option('--permit-ecc-key-import-functions'
mis @option('--processor-management-enabled'
mis @option('--reserve-resources'
mis @option('--short-name'
mis @option('--ssc-ipv6-gateway'
```

**Comment by andy-maier** [comment]
The difference with the latest commit 49c16f6ee is now smaller, but not zero:

partition create:
```
mis @option('--access-coprocessor-group-set'
mis @option('--cp-processing-weight-capped'
mis @option('--ifl-processing-weight-capped'
mis @option('--processor-management-enabled'
```
partition update:
```
mis @option('--access-coprocessor-group-set'
mis @option('--boot-iso-insfile'
mis @option('--boot-timeout'
mis @option('--processor-management-enabled'
```

**Comment by andy-maier** [comment]
On the `--access-coprocessor-group-set` option: I think we should remove the option if the property does not have a function.

File: zhmccli/_cmd_partition.py (hunk 1 above)
Comment (andy-maier) [reply zhmccli/_cmd_partition.py:371]: agreed

**Comment by andy-maier** [comment]
I have verified by comparing with the state before PR #492 that all options except `--access-coprocessor-group-set` are now available again.

**Review by andy-maier — APPROVED** [review]

**Pull request merged**